Sources/StreamChatSwiftUI/ChatChannel/Reactions/ReactionUserView.swift

```diff
@@ -0,0 +1,155 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+/// View displaying single user reaction.
+struct ReactionUserView: View {
+    
+    @Injected(\.chatClient) private var chatClient
+    @Injected(\.fonts) private var fonts
+    
+    var reaction: ChatMessageReaction
+    var imageSize: CGFloat
+    
+    private var isCurrentUser: Bool {
+        chatClient.currentUserId == reaction.author.id
```

Comment:
That's the only place we use the `chatClient`. If we injected the `isCurrentUser` via a variable, we would directly have previews available here. I know that the caller also doesn't have `chatClient` available, so I don't think we should change anything here...
just thinking about what we would need to do to have previews :)

Reply:
Also the author (`ChatUser`) inside the `ChatMessageReaction` is core data object, so that one might be a problem for previews.